README.md

```diff
@@ -52,6 +52,7 @@ Archipelago welcomes and appreciates any type of contribution, from use cases an
 ## Caring & Coding + Fixing
 * [Diego Pino](https://github.com/DiegoPino)
 * [Giancarlo Birello](https://github.com/giancarlobi)
+* [Allison Lund](https://github.com/alliomeria)
```

Comment:
@DiegoPino and done!

Reply:
Wonderful. Happy to see your name there. We may want to talk about the order some other day =)